app/src/main/java/org/mozilla/fenix/tabtray/FenixTabsAdapter.kt

```diff
@@ -70,6 +70,10 @@ class FenixTabsAdapter(
         (holder as TabTrayViewHolder).updateAccessibilityRowIndex(holder.itemView, newIndex)
 
         holder.tab?.let { tab ->
+            val shouldBeChecked =
```

Comment:
We could probably move `shouldBeChecked` into `showCheckedIfSelected` and simplify the code further.

```
 private fun showCheckedIfSelected(holder: TabViewHolder) {
     val shouldBeChecked =
        mode is TabTrayDialogFragmentState.Mode.MultiSelect && selectedItems.contains(holder.tab)	            mode is TabTrayDialogFragmentState.Mode.MultiSelect && selectedItems.contains(holder.tab)
     view.checkmark.isVisible = shouldBeChecked
     view.selected_mask.isVisible = shouldBeChecked
}
```